app/assets/javascripts/packages.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Get in the habit of deleting files you don't use. Rails can be really heavy and comes with a lot of features you won't need. Keep your workspace tidy & it can potentially speed up performance.